### PR TITLE
Fix mini-css-extract-plugin types to work with Webpack 5

### PR DESCRIPTION
**Pull request by xarety**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [link](https://webpack.js.org/plugins/mini-css-extract-plugin/)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/mini-css-extract-plugin/index.d.ts
Comment (peterblazejewicz) [inline types/mini-css-extract-plugin/index.d.ts:39]: 👍

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@xarety thx!

**Comment by xarety** [comment]
@typescript-bot test

**Comment by xarety** [comment]
Ready to merge

**Comment by xarety** [comment]
Hey, @basarat, could you approve these changes? There no changes for the tests as the external API wasn't changed.

**Comment by xarety** [comment]
Hey, @RyanCavanaugh, how can I merge this PR?

**Comment by peterblazejewicz** [comment]
Add test coverage, otherwise patiently wait for additional review by DT maintainer,

**Comment by starikcetin** [comment]
I need these. How can I add the tests? Or what are we testing exactly?

**Comment by xarety** [comment]
Closed in favor #49318

**Pull request closed without merging**
